docs/content/develop/step-1a.md

```diff
@@ -19,12 +19,29 @@ npm install
 
 ## Generating a PatternFly Element
 
-Using the [generator-pfelement](https://github.com/patternfly/generator-pfelement), which is installed as a dev dependency, the generator will ask you a few questions that will help with the scaffolding. Make sure you are in the root directory of the PatternFly Elements repository.
+The easiest way to build a new component from scratch is to use the generator. The generator will ask you a few questions that will help with the scaffolding. Make sure you are in the root directory of the PatternFly Elements repository.
 
 ```
 npm run new
 ```
 
+When creating your new components, you may find you are entering the same answers over and over again.  To prevent this, you can add a `project.conf.json` file with some of your preferences stored.  Here is a template for that file:
+
+```json
```

Comment:
@mwcz @kylebuch8 Want to bring y'alls attention to this doc section I added since this will take care of being able to set defaults.  If type is not set in this file Kyle, then the prompt will ask if you want to create a standalone or a pfelement.